charts/plugin-br-pix-switch/templates/plugin-br-pix-switch/configmap.yaml

```
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "plugin-br-pix-switch.fullname" . }}
+  labels:
+    {{- include "plugin-br-pix-switch.labels" (dict "context" . "name" .Values.pixSwitch.name ) | nindent 4 }}
+data:
+  # Application
+  ENV_NAME: {{ .Values.pixSwitch.configmap.ENV_NAME | default "development" | quote }}
+  VERSION: {{ .Values.pixSwitch.configmap.VERSION | default "1.0.0-beta.1" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)